Signing/node-express/src/criipto-operations.ts

```diff
@@ -52,21 +52,28 @@ export async function getSignatureOrder(id: string, includeDocuments?: boolean){
   return signatureOrder;
 }
 
-export async function getOrders(first: number) {
-  console.log('Getting orders ...');
-  const orders = await client.querySignatureOrders({first})
-  .then((result) => {
-    return result
-    .slice(-10)
-    .map((order) => ({
-      id: order.id,
-      status: order.status,
-      title: order.title || 'Untitled',
-    }));
-  });
+export async function getOrdersByStatus(first: number, status?: NonNullable<Parameters<typeof client["querySignatureOrders"]>[0]>["status"]) {
+  console.log(`Getting orders ...`);
+  const orders = await client.querySignatureOrders({ first, status })
+    .then((result) => {
+      return result
+        .slice(-10)
+        .map((order) => ({
+          id: order.id,
+          status: order.status,
+          title: order.title || 'Untitled',
+        }));
+    });
   return orders;
 }
 
+export async function getOrders(first: number) {
+  const openOrders = await getOrdersByStatus(first, "OPEN");
+  const closedOrders = await getOrdersByStatus(first, "CLOSED");
+  const allOrders = await getOrdersByStatus(first);
+  return { openOrders, closedOrders, allOrders };
```

Comment:
This will execute 3 serial calls to the API each time someone wants to access just part of the data.
It makes more sense for the call sites to just call `getOrdersByStatus` directly.